leaflet-maplibre-gl.d.ts

```diff
@@ -1,9 +1,12 @@
 import * as L from 'leaflet';
 import { Map, MapboxOptions } from 'maplibre-gl';
 
+
 declare module 'leaflet' {
+    type LeafletMaplibreGLOptions = Omit<MapboxOptions, "container">;
```

Comment:
Can you describe why `container` is problematic?

Reply:
This plugin creates its own container (being a Leaflet layer) and passes this container to the Maplibre-GL instance.
See: https://github.com/maplibre/maplibre-gl-leaflet/blob/main/leaflet-maplibre-gl.js#L120

Passing a `container` property would simply be ignored. However, the type (MapboxOptions) defines `container` as a required property which is not the case when passing options to this Leaflet plugin. This is why the type of the options you can pass must change.